src/main/java/seedu/address/model/person/Person.java

```diff
@@ -2,10 +2,7 @@
 
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Some advice, you should configure your IDE not to do this in settings -  https://se-education.org/guides/tutorials/intellijCodeStyle.html